mobile-app/src/screens/ArtworkScreen.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import { Text, View, StyleSheet } from 'react-native';
+import { Title } from '../components/Title';
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
     alignItems: 'center',
-    justifyContent: 'center',
+    paddingTop: 45,
   },
 });
 
 const ArtworkScreen: React.FC = () => (
+  
   <View style={styles.container}>
+    <Title text="Artwork Screen"/>
```

Comment:
```suggestion
  <View style={styles.container}>
    <Title text="Artwork Screen" />
```

Probably the most nit-picky suggestion on earth :)